### DISCUSSION TITLE
Username instead of Hostname

### DISCUSSION THREAD

**Discussion by esgheorghe** [Ylianst/MeshCentral#6526] [Q&A]
Hello,

Hello,

Is it possible to display the username instead of the hostname in a group that is filled with the MeshCentral Assistant or / and to extend the search so that it also searches for the logged-in users?

We want to connect in Citrix sessions, which works well, but the search becomes very difficult with several 100 users if only the servername is displayed.

Many thanks & BR
Emilian-Stefan

Sample:
![grafik](https://github.com/user-attachments/assets/7199500f-724e-46bd-b11d-b426173eb823)

**Comment by si458** [comment]
the user is already displayed as you have selected the `user` column from the cog option, or is this not what you mean?
(can u draw pic/example what you mean if i mis-understood?)

also you can already search for a certain user by using the search filters
https://ylianst.github.io/MeshCentral/meshcentral/devicetabs/#search-or-filter
`u:user1` `user:bob`

PS: nice to see you testing the new bootstrap theme hehe

**Comment by esgheorghe** [comment]
Hi,

thank you very much for the quick response.
The search should search for the user without entering u: first. We have 
many power users who need to provide support for certain programs but 
are not IT related.

Here is a screenshot of how we envision this, but only for a specific 
group.

Yes, the new design is great :-)
BR
Emilian-Stefan

Am 2024-11-11 14:41, schrieb Simon Smith:

> the user is already displayed as you have selected the user column from 
> the cog option, or is this not what you mean?
> (can u draw pic/example what you mean if i mis-understood?)
> 
> also you can already search for a certain user by using the search 
> filters
> https://ylianst.github.io/MeshCentral/meshcentral/devicetabs/#search-or-filter
> u:user1 user:bob
> 
> PS: nice to see you testing the new bootstrap theme hehe
> 
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].
> You are receiving this because you authored the thread.Message ID: 
> ***@***.***>


Links:
------
[1] 
https://github.com/Ylianst/MeshCentral/discussions/6526#discussioncomment-11214863
[2]

**Reply by si458** [reply]
no picture attached but i understand what you mean!
we would have to change the search method so would require a little work,
so if you wanted it to search all the filters instead of you typing them, open a github enhancement request 👍
